staging/src/k8s.io/client-go/tools/record/util/util.go

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
@yastij - This refactoring (moving those functions to a separate file) is non-controversial at all.
Feel free to move it to a separate PR and I'm happy to approve it as is now (to reduce size of this change).

Reply:
sure.